math/src/stableswap/math.rs

```diff
@@ -21,69 +21,89 @@ const PRECISION: u8 = 1;
 /// D - number of iterations to use for Newton's formula to calculate parameter D ( it should be >=1 otherwise it wont converge at all and will always fail
 /// Y - number of iterations to use for Newton's formula to calculate reserve Y ( it should be >=1 otherwise it wont converge at all and will always fail
 pub fn calculate_out_given_in<const D: u8, const Y: u8>(
-	balances: &[AssetReserve],
+	initial_reserves: &[AssetReserve],
```

Comment:
`initial_reserves` could be bit misleading as that points out to the past time, but you want current state, i would just call it `reserves` or `current_reserves` if you want to be more exact

Reply:
i see you call it reserves later anyway :) so keep it consistent